src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -201,6 +201,20 @@ You can pass many different options to tracing, but it comes with reasonable def
 - If available: Browser Resources (Scripts, CSS, Images ...)
 - If available: Browser Performance API Marks
 
+*tracingOrigins Option*
+ 
+The default value of `tracingOrigins` is : `new ApmIntegrations.Tracing({tracingOrigins: ['localhost', /^\//]})`. This options will attach the `sentry-trace` header to all outgoing XHR/fetch requests that contain a string in the list or match the regex. In case your frontend is making requests to a different domain you might want to add it there so we attach the `sentry-trace`.
```

Comment:
```suggestion
The default value of `tracingOrigins` is : `new ApmIntegrations.Tracing({tracingOrigins: ['localhost', /^\//]})`. The JavaScript SDK will attach the `sentry-trace` header to all outgoing XHR/fetch requests that contain a string in the list or match a regex. In case your frontend is making requests to a different domain you might want to add it there in order to propagate the `sentry-trace` header to the backend services, required to link transactions together as part of a single trace.
```

Reply:
Suggestion to the suggestion: `, required` should prob be something like `, which is required`